packages/server/api/routes/reviews.router.js

```diff
@@ -62,4 +62,17 @@ router.delete('/:id', (req, res) => {
   });
 });
 
+router.post('/', (req, res) => {
+  const { movieId, userId, rating, reviewText } = req.body;
+
+  reviewsController
+    .addReview(movieId, userId, rating, reviewText)
+    .then(() => {
+      res.sendStatus(201);
+    })
+    .catch((error) => {
+      res.status(500).json({ error: error.message });
+    });
```

Comment:
I am unsure how your express error catching is configured, but I am pretty sure that although in controller you throw a bunch of 400s in specific cases, your rest api will only throw this 500 no matter happens :)

Depending how the express error handling is configures, here you probably should include something like `res.status(error.status).json(({error: error.message})`